scripts/ovs/templates/small/setup.json

```diff
@@ -0,0 +1,711 @@
+{
+  "ci":{
+    "setup": true,
+    "validation": false,
+    "scenarios": false,
+    "cleanup": false,
+    "send_to_testrail": false,
+    "fail_on_failed_scenario": false,
+    "setup_retries": 1,
+    "user": {
+      "shell": {
+        "username": "root",
+        "password": "foobar"
+      },
+      "api": {
+        "username": "admin",
+        "password": "admin"
+      }
+    },
+    "grid_ip": "10.100.50.18"
+  },
+  "setup": {
+    "domains": [],
+    "backends": [
+      {
+        "name": "cachebackend01",
+        "domains": {"domain_guids": []},
+        "scaling": "LOCAL",
+        "presets": [ ],
+        "osds": {
+          "{{cpu_01_ip}}": {
+            "{{cpu_01_disk_1}}": 2,
+            "{{cpu_01_disk_2}}": 2
+          },
+          "{{cpu_02_ip}}": {
+            "{{cpu_02_disk_1}}": 2,
+            "{{cpu_02_disk_2}}": 2
+          },
+          "{{cpu_03_ip}}": {
+            "{{cpu_03_disk_1}}": 2,
+            "{{cpu_03_disk_2}}": 2
+          },
+          "{{cpu_04_ip}}": {
+            "{{cpu_04_disk_1}}": 2,
+            "{{cpu_04_disk_2}}": 2
+          },
+          "{{cpu_05_ip}}": {
+            "{{cpu_05_disk_1}}": 2,
+            "{{cpu_05_disk_2}}": 2
+          },
+          "{{cpu_06_ip}}": {
+            "{{cpu_06_disk_1}}": 2,
+            "{{cpu_06_disk_2}}": 2
+          },
+          "{{cpu_07_ip}}": {
+            "{{cpu_07_disk_1}}": 2,
+            "{{cpu_07_disk_2}}": 2
+          },
+          "{{cpu_08_ip}}": {
+            "{{cpu_08_disk_1}}": 2,
+            "{{cpu_08_disk_2}}": 2
+          }
+        }
+      },
+      {
+        "name": "cachebackend02",
+        "domains": {"domain_guids": []},
+        "scaling": "LOCAL",
+        "presets": [ ],
+        "osds": {
+          "{{cpu_01_ip}}": {
+            "{{cpu_01_disk_3}}": 2,
+            "{{cpu_01_disk_4}}": 2
+          },
+          "{{cpu_02_ip}}": {
+            "{{cpu_02_disk_3}}": 2,
+            "{{cpu_02_disk_4}}": 2
+          },
+          "{{cpu_03_ip}}": {
+            "{{cpu_03_disk_3}}": 2,
+            "{{cpu_03_disk_4}}": 2
+          },
+          "{{cpu_04_ip}}": {
+            "{{cpu_04_disk_3}}": 2,
+            "{{cpu_04_disk_4}}": 2
+          },
+          "{{cpu_05_ip}}": {
+            "{{cpu_05_disk_3}}": 2,
+            "{{cpu_05_disk_4}}": 2
+          },
+          "{{cpu_06_ip}}": {
+            "{{cpu_06_disk_3}}": 2,
+            "{{cpu_06_disk_4}}": 2
+          },
+          "{{cpu_07_ip}}": {
+            "{{cpu_07_disk_3}}": 2,
+            "{{cpu_07_disk_4}}": 2
+          },
+          "{{cpu_08_ip}}": {
+            "{{cpu_08_disk_3}}": 2,
+            "{{cpu_08_disk_4}}": 2
+          }
+        }
+      },
+      {
+        "name": "backend01",
+        "domains": {"domain_guids": []},
+        "scaling": "LOCAL",
+        "presets": [ ],
+        "osds": {
+          "{{storage_01_ip}}": {
```

Comment:
this should be more dynamic loop over storage nodes and use hdd / 2 disks (first half)